### PR TITLE
Encodings for paper

### PR DESCRIPTION
**Pull request by susuhahnml**

### CODE REVIEW

File: asp_interactive_configuration/examples/metro/metro_v1.png
Comment (rtaupe) [inline asp_interactive_configuration/examples/metro/metro_v1.png:1]: The font looks nicer in the new version! But why are the class names not capitalized anymore?

File: asp_interactive_configuration/ooasp/config.py [hunk 1]
```diff
@@ -23,7 +24,7 @@ class  OOASPConfiguration:
             UNIFIERS (Namespace): All clorm unifiers (classes) used to link objects with predicates
     """
 
-    def __init__(self, name:str, kb: OOASPKnowledgeBase):
+    def __init__(self, name:str, kb: OOASPKnowledgeBase, simplifyed_encodings=False):
```
Comment (rtaupe) [inline asp_interactive_configuration/ooasp/config.py]: simplifyed -> simplified ?

File: asp_interactive_configuration/ooasp/interactive.py [hunk 2]
```diff
@@ -85,7 +85,7 @@ class InteractiveConfigurator:
             last_size_grounded (int): The last size that was grounded
     """
 
-    def __init__(self, kb:OOASPKnowledgeBase, config_name:str, additional_files:List=None, additional_prg:str=""):
+    def __init__(self, kb:OOASPKnowledgeBase, config_name:str, additional_files:List=None, additional_prg:str="", simplifyed_encodings=False):
```
Comment (rtaupe) [inline asp_interactive_configuration/ooasp/interactive.py]: simplifyed -> simplified ?

File: asp_interactive_configuration/ooasp/kb.py [hunk 3]
```diff
@@ -20,7 +21,7 @@ class  OOASPKnowledgeBase:
             UNIFIERS (Namespace): All clorm unifiers (classes) used to link objects with predicates
     """
 
-    def __init__(self, name:str):
+    def __init__(self, name:str, simplifyed_encodings=False):
```
Comment (rtaupe) [inline asp_interactive_configuration/ooasp/kb.py]: simplifyed -> simplified ?

File: asp_interactive_configuration/ooasp/tests/test_performance.py [hunk 4]
```diff
@@ -4,31 +4,31 @@
 from ooasp.interactive import InteractiveConfigurator
 from ooasp.kb import OOASPKnowledgeBase
 
-def test_solve_multiple_elements():
-    """ create elements and solve under 30 seconds """
+# def test_solve_multiple_elements():
```
Comment (rtaupe) [inline asp_interactive_configuration/ooasp/tests/test_performance.py]: why is (almost) everything commented-out in this file?

**Review by rtaupe — CHANGES_REQUESTED** [review]
Thank you for your changes!
Please let's just address my minor comments before merging.

File: asp_interactive_configuration/examples/metro/metro_v1.png
Comment (susuhahnml) [reply asp_interactive_configuration/examples/metro/metro_v1.png:1]: It must have been removed in some update. I will put that back :)

File: asp_interactive_configuration/ooasp/tests/test_performance.py (hunk 4 above)
Comment (susuhahnml) [reply asp_interactive_configuration/ooasp/tests/test_performance.py]: I must have commented it out while running the tests multiple times. I will uncomment

**Comment by susuhahnml** [comment]
@rtaupe Done! :)

**Review by rtaupe — APPROVED** [review]

**Pull request merged**